samples/dnn/object_detection.cpp

```diff
@@ -283,27 +313,32 @@ int main(int argc, char** argv)
 
     // Process frames.
     Mat frame, blob;
-    while (waitKey(1) < 0)
+    while (waitKey(100) < 0)
     {
         cap >> frame;
         if (frame.empty())
         {
             waitKey();
             break;
         }
+        //![preprocess_call_func]
+        preprocess(frame, net, Size(inpWidth, inpHeight));
```

Comment:
any reason to use global preprocessing parameters and not sending them through the function?